cypress/e2e/login.test.ts

```diff
@@ -1,13 +1,36 @@
-describe("Navigation Test", () => {
+describe("Homepage test", () => {
 	beforeEach(() => {
 		// TODO: Add database seeding for e2e test user instead of using local profiles.
 		cy.login();
 		// Visit a route in order to allow cypress to actually set the cookie
 		cy.visit("/");
 	})
 
-	it("can direct to coachees", () => {
-		cy.visit("/coachees");
-		cy.findAllByText("资深导师职责").should("be.visible");
-	});
+	it("can render home page", () => {
+		cy.findAllByText("我的会议").should("be.visible");
+	})
+
+	it("can join meeting", () => {
+		cy.get('.css-1r08f2c').each(($ele) => {
```

Comment:
I'm not sure if you should rely on this css ID. It is auto generated and I'm not aware of a guarantee that it stays the same across builds.

Reply:
done